presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -48,6 +48,7 @@ services:
       - ${PRESTO_JDBC_DRIVER_JAR}:/docker/volumes/jdbc/driver.jar
       - ../../../conf/tempto/tempto-configuration-for-docker-default.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
       - ../../../target/test-reports:/docker/volumes/test-reports
+      - ~/.presto_history:/root/.presto_history
```

Comment:
If this file does not exist initially, will `~/.presto_history` created as owned by `root` ?

Reply:
if yes, you can use eg `${TMP}/$(whoami)-presto-cli-history-docker` instead